script.js

```diff
@@ -1,12 +1,132 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
-}
-
-function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
-}
+const rootElem = document.getElementById("root");
+const footerEl = document.createElement("footer");
+const paragraphEl = document.createElement("p");
+const footerLinkEl = document.createElement("a");
+
+paragraphEl.innerText =
+  "Please note that the data has (originally) been pulled in from: ";
+footerLinkEl.setAttribute("href", "https://www.tvmaze.com/api#licensing");
+footerLinkEl.setAttribute("target", "_blank");
+footerEl.classList.add("footer-root");
+footerLinkEl.innerText = "TV MAZE";
+
+footerEl.append(paragraphEl);
+paragraphEl.appendChild(footerLinkEl);
+document.body.appendChild(footerEl);
+
+let allEpisodes = getAllEpisodes();
+const allEpisodesCounter = getAllEpisodes().length;
+
+const setup = () => {
+  allEpisodes = getAllEpisodes();
+  getEpisodeOfTvShows(allEpisodes);
+};
+
+// Render each episode to the website //
+const renderEpisodeToWebsite = (name, season, number, image, summary) => {
+  divCardH1El.innerText = `${name} - S${season
+    .toString()
+    .padStart(2, "0")}E${number.toString().padStart(2, "0")}`;
+  imgCardEl.src = `${image.medium}`;
+  divCardBodyEl.innerHTML = `${summary}`;
+
+  rootElem.append(divCardEl);
+};
+
+// Create all episodes of tv shows //
+const getEpisodeOfTvShows = (episodeList) => {
+  rootElem.innerHTML = "";
+
+  episodeList.forEach(({ name, season, number, image, summary }) => {
+    divCardEl = document.createElement("div");
+    divCardEl.classList.add("card");
+    divCardH1El = document.createElement("h1");
+    divCardH1El.classList.add("card-header");
+    imgCardEl = document.createElement("img");
+    imgCardEl.classList.add("card-img-top");
+    imgCardEl.setAttribute("alt", `A image from ${name}`);
+    divCardBodyEl = document.createElement("div");
+    divCardBodyEl.classList.add("card-body");
+    divCardEl.appendChild(divCardH1El);
+    divCardEl.appendChild(imgCardEl);
+    divCardEl.appendChild(divCardBodyEl);
+
+    renderEpisodeToWebsite(name, season, number, image, summary);
+  });
+
+  // Add and wait for action to remove the watched episode by clicking on corresponding image //
+  let imageEl = document.getElementsByClassName("card-img-top");
+  Array.from(imageEl).forEach((image) => {
+    image.addEventListener(
+      "click",
+      (e) => {
+        let removeTarget = e.target.parentNode;
+        removeTarget.parentNode.removeChild(removeTarget);
```

Comment:
Would be nice to have this tied to a remove button instead of clicking the image. When I click the Image im more expecting the website to take me to a page with just that episode.